__tests__/wallet/wallet.test.ts

```diff
@@ -325,3 +324,111 @@ test('generateCreateWalletAuthData should return correct auth data', async () =>
   expect(xpubMessage.verify(xpubAddress, authData.xpubkeySignature)).toBe(true);
   expect(authXpubMessage.verify(authXpubAddress, authData.authXpubkeySignature)).toBe(true);
 });
+
+test('registerDeviceToPushNotification', async () => {
+  const requestPassword = jest.fn();
+  const network = new Network('testnet');
+  const seed = 'purse orchard camera cloud piece joke hospital mechanic timber horror shoulder rebuild you decrease garlic derive rebuild random naive elbow depart okay parrot cliff';
```

Comment:
Since the seed used is the same, how about creating a constant for it and reusing on every test?